pkg/ansible/controller/controller.go

```diff
@@ -16,14 +16,13 @@ package controller
 
 import (
 	"fmt"
-	"log"
 	"strings"
 	"time"
 
 	"github.com/operator-framework/operator-sdk/pkg/ansible/events"
 	"github.com/operator-framework/operator-sdk/pkg/ansible/runner"
 
-	"github.com/sirupsen/logrus"
+	log "github.com/sirupsen/logrus"
```

Comment:
I'm wondering if we should use the `logr` interface in the ansible (and eventually helm) operator packages and main functions.

If those operators had been developed using the scaffolding in this PR, they would likely be using logr-based SDK log package.

Also, in the case where a user migrates from ansible/helm to a go-based hybrid, would their operator logging be a mix of logr (from any added controllers) and logrus (from `pkg/ansible`)?

Reply:
@mhrivnak @shawn-hurley what do you think should be done with `pkg/ansible/events`? There's probably a similar approach with `logr` to get the same functionality.

Reply:
Re-implementing with `logr`.

Reply:
@estroz thank you for implementing this for the pkg/ansible!